business_requirement/models/business.py

```diff
@@ -269,7 +270,8 @@ def name_get(self):
         result = []
         for br in self:
             if br.ref:
-                formatted_name = u'[{}] {}'.format(br.ref, br.description)
+                formatted_name = u'[{}][{}] {}'.format(br.ref, br.name,
```

Comment:
not sure here but if the requirement is fulfilled, this is fine by me.

I want: [WBS][name] Description or [name] Description (if WBS empty)